test/__file_snapshots__/esmodules--async-await-0

```diff
@@ -1,5 +1,3 @@
-import _asyncToGenerator from "@babel/runtime-corejs2/helpers/esm/asyncToGenerator";
-
-_asyncToGenerator(function* () {
-  yield foo();
+(async function () {
+  await foo();
```

Comment:
I'm curious, can we add a test for `async`/`await` w/ `safari10` as a target ?

Reply:
Good point!

Since we serve the `esmodules` bundles to Safari 10, it will get this change.  

Looks like the issue with Safari 10 and async functions is linked to arrow functions, and if we set-up terser properly we should be fine.


- https://github.com/babel/preset-modules#important-minification  
- https://github.com/babel/preset-modules/blob/master/src/plugins/transform-async-arrows-in-class/index.js
- https://caniuse.com/#search=async

Reply:
Suggested change:

```
              new TerserPlugin({
                terserOptions: {
                  ecma: ES2017 ? '2017' : '5',
                  safari10: true,
                  compress: {
                    warnings: false,
                  },
                  output: {
                    comments: false,
                  },
                },
                parallel: true,
                cache: true,
                sourceMap: true,
              }),
```